src/components/ItemDescription.js

```diff
@@ -8,15 +8,11 @@ function ItemDescription(props) {
   const country = props.country
```

Comment:
- [ ] The `ItemDescription` component is a stateful component that deals with the redux store. So basically it is a `container` component. I recommend you create a `container` folder inside the `src` directory and move this file to the `container` folder for a more logical document structure. :pray: